weaviate/helm/weaviate/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: weaviate
 description: helm chart for weaviate
 type: application
-version: 0.1.1
```

Comment:
do you need to modify the chart deps.yaml here?

Reply:
No all good